### ISSUE TITLE
dry/wet response curve for the ambiance Fx

### ISSUE THREAD

**Issue by rootapprox** [Befaco/Oneiroi#20]
would be great to have the dry/wet response curve to equal power crossfading (no volume drop at 100%)
so the sounds are not lost in the effect with full wet specially with extreme settings of spacetime

**Comment by hirnlego** [comment]
@rootapprox please, check if the new beta 8 improves this.

**Comment by rootapprox** [comment]
@hirnlego Yes , sounds very great now ! thanks a lot!!

**Comment by athousanddetails** [comment]
> @hirnlego Yes , sounds very great now ! thanks a lot!!
> 
> can i have a suggestion to have the SINE / SS+WT less hot in volume when the faders full up

YES, @hirnlego this needs some attention... the volumes are too hot. You easily get distortion at the filter due to Sine / SS / WT...
